File: articles/feedback-loops-in-algorithmic-systems/c/feedback_risk.c
#include <stdio.h>

int main(void) {
    double amplification = 0.82;
    double concentration = 0.76;
    double intervention = 0.44;
    double drift = 0.28;
    double recursive_data = 0.31;
    double score = (amplification + concentration + intervention + drift + recursive_data) / 5.0;
    printf("feedback_risk_score=%.4f\n", score);
    return 0;
}

File: articles/feedback-loops-in-algorithmic-systems/go/feedback_risk.go
package main

import "fmt"

func main() {
	amplification := 0.82
	concentration := 0.76
	intervention := 0.44
	drift := 0.28
	recursiveData := 0.31
	score := (amplification + concentration + intervention + drift + recursiveData) / 5.0
	fmt.Printf("feedback_risk_score=%.4f\n", score)
}
